### ISSUE TITLE
Analyze: Fix Headers Comparison when Multiple Orders are Possible

### ISSUE THREAD

**Issue by blakebyrnes** [ulixee/double-agent#52]
We seem to have an issue when multiple orders are possible for a header order. The introduction of the sec-ca-ua headers in chrome appears to have introduced entropy to the header order within a single session, even with all other variables remaining constant (I haven't verified in chromium source - possibly this is intentional).

We need to find a better strategy for comparing the patterns when 1 of many patterns is a valid match.

**Comment by knaitas** [comment]
Would be nice to launch and test this if the packages weren't missing from NPM :(

**Comment by blakebyrnes** [comment]
@knaitas Yes, we're working to release Hero! In interim, you can just download the Hero project and use it that way. Should have Hero out in next few weeks though.
